src/Main.cxx

```diff
@@ -106,12 +104,12 @@ Main::Main(MainOpt &opt)
     }
   }
   curl = ::make_unique<stub_curl>();
-  if (not main_opt.status_uri.host.empty()) {
+  if (not main_opt.StatusReportURI.host.empty()) {
```

Comment:
the other 'if not empty' statements use `!` rather than `not`, might make sense to just use one or the other throughout the code. I'm not sure what the LLVM standard prefers but i'd guess it'd be the `!` as they use it in some of their sample code.